guides/common/modules/proc_enabling-structured-apt-content.adoc

```diff
@@ -0,0 +1,52 @@
+[id="enabling-structured-apt-content"]
+= Enabling structured APT content
+
+You can enable structured APT content on your {Project} to use the same Deb repository structure as the remote repositories they are synchronized from.
+
+.Prerequisites
+* The host is registered to {ProjectName}.
+include::snip_prerequisite-project-client-repository-enabled.adoc[]
```

Comment:
What do you need the Client repo for? Again, confused. It isn't clear from the procedure.